CONTRIBUTING.md

```diff
@@ -67,6 +67,16 @@ For development you are going to want to checkout out the source. To get it, clo
     cd linguist/
     script/bootstrap
 
+To run Linguist from the cloned repository directory, you will need to generate the code samples first:
+
+    bundle exec rake samples
+
+If you are not running the tests below, re-run this command every time you modify any of the [samples][samples].
```

Comment:
> *"If you are not running the tests ~~below~~"*

Unless there's a need to be specific, and unless there's another set of tests in Linguist which aren't relevant to the current paragraph, we can 🔥  that.

> "re-run this command every time you modify any of the [samples][samples]."

This sentence feels a bit... clunky to me. I'd word that like this:

> "run this command each time a [sample] has been modified:"